### PR TITLE
mkt-v2 content added

### PR DESCRIPTION
**Pull request by ayo-klaytn**
## What it solves
new content on payment splitter, nft royalty, nft with operator filter added

### CODE REVIEW

File: docs/klaytn-contracts/payment-splitter/introduction.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+The Payment Splitter contract enables a group of people to split payments (native tokens and ERC20 tokens) in a secure and transparent manner. This split is made possible by providing the *payee* and the *share* in the contract. The payee is the person to whom you want to pay a certain amount, and the share is the percentage amount the payee will receive. It should be noted that the shares can be distributed in any random ratio or in equal parts.
+
+With the use of payment splitter contracts, 
+* The payee does not have to remind the payer of the amount owed on a regular basis.
+* The payee cannot withdraw more than they are owed
+* Transparency is promoted among team members.
+
+## How it works <a id="How it works"></a> 
+
+Each payee will be able to claim the amount due them from all payments received by the contract. The payment splitter contract follows the [pull method](https://river.com/learn/pull-system-vs-push-system/), which means that payee can safely pull funds out of the contract by calling the release function.
```
Comment (praveen-klaytn) [inline docs/klaytn-contracts/payment-splitter/introduction.md:21]: could you please rephrase this "claim the amount due them from all payments"

File: docs/klaytn-contracts/payment-splitter/sdks-and-tutorials.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+    })
+    .catch(err => {
+        console.log(err);
+    })
+}
+
+// function to pull payee's share from the contract
+// Note anyone can call this function provided it is the payee's address
+const releaseToPayee = async () => {
+    const payeeAddress =  “<PASTE PAYEE ADDRESS>”;
```
Comment (praveen-klaytn) [inline docs/klaytn-contracts/payment-splitter/sdks-and-tutorials.md:135]: could you please move below statement to the top as the values are hardcoded

const payeeAddress = "<PASTE PAYEE ADDRESS>";

File: docs/klaytn-contracts/payment-splitter/sdks-and-tutorials.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+const Caver = require('caver-js')
+const contractABI = require("../abi/paymentSplitter.json")
+
+// Initialize caver.js and the paymentSplitter contract
+const caver = new Caver('https://api.baobab.klaytn.net:8651/')
+const contractAddr = "paste contract address"
+const contract = caver.contract.create(contractABI, contractAddr);
+
+// Create and add keyring
+const keyring = caver.wallet.keyring.createFromPrivateKey('<Paste private key from Kaikas Wallet>')
```
Comment (praveen-klaytn) [inline docs/klaytn-contracts/payment-splitter/sdks-and-tutorials.md:116]: extract private key from here please

File: docs/klaytn-contracts/payment-splitter/sdks-and-tutorials.md [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+    contract.methods.released(addr).call()
+    .then(result => {
+        console.log(result);
+    })
+ }
+
+
+mintTokens();
+// releaseToPayee();
+// getAmountReleased("<PASTE PAYEE ADDRESS>");
```
Comment (praveen-klaytn) [inline docs/klaytn-contracts/payment-splitter/sdks-and-tutorials.md:157]: extract payee address to a constant, declare at the start and pass the variable to the method.

File: docs/klaytn-contracts/nft-royalty/introduction.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+
+## Definition of NFT Royalties <a id="Definition of NFT Royalties"></a>
+NFT royalties are payments made to the original creator of a digital asset for each time that asset is resold. This means that when this asset is sold, a certain percentage is paid to the creator as a reward for the asset's value.
+
+For example, a creator may have published an asset on the Marketplace and initially sold it for 10,000 KLAY. Simultaneously, the creator included a 10% NFT royalty percentage for subsequent sales. Thus, if the buyer resells and earns 100,000 KLAY from the resale, the original creator will receive 10% (10,000 KLAY).
+
+Isn't that cool? NFT royalties let creators passively earn from their work.
+
+## How does it work? <a id="How does it work"></a>
+When creatives use NFT Royalty for subsequent sales, they will continue to earn money from their work. The creator usually specifies the royalty policy at the platform or on-chain level.
```
[the remark this answers is not in the document]
Comment (ayo-klaytn) [reply docs/klaytn-contracts/nft-royalty/introduction.md:18]: Yeah literally creatives means a person whose job involves creative work like work of art .

[the remark this answers is not in the document]
Comment (praveen-klaytn) [reply docs/klaytn-contracts/nft-royalty/introduction.md:18]: please ignore this then. Thanks

**Comment by ayo-klaytn** [comment]
@praveen-klaytn updated the code to contain the review suggestions. Kindly take a look
Thanks

**Review by praveen-klaytn — APPROVED** [review]

**Pull request merged**